spec/helpers/ad_renewal_letters_exports_helper_spec.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe AdRenewalLettersExportsHelper, type: :helper do
+  describe "#format_email_to_name" do
+    it "takes an email and format it into a new string" do
+      expect(helper.format_email_to_name("super_agent@wex.gov.uk")).to eq("Super Agent")
```

Comment:
Could we change the example to use a period?

```suggestion
      expect(helper.format_email_to_name("katherine.johnson@nasa.org.uk")).to eq("Katherine Johnson")
```

We seed our test data with names that use underscores, but in the actual app 99% of the time the time it'll be a period.

Also is it worth having a scenario that covers emails without any separator, just to ensure we don't get an exception if we encounter one?

N.B. See comment on the helper itself, as I'm not sure it will work if the email uses a `.`.